chapter_16_javascript.asciidoc

```diff
@@ -901,7 +914,7 @@ And in our tests file, we call `initialize()` in our key test:
 });
 ----
 ====
-
+//RITA: What does "here" refer to?
```

Comment:
it runs on from the last (non-code) sentence:  

> And in our tests file, we call `initialize()` in our key test:

the "here" shows the line where we call "initialize()".

Reply:
For the sake of clarity, would you mind adding "Here's where we call initialize()"?